lib/Driver.cpp

```diff
@@ -63,25 +63,34 @@ std::vector<std::unique_ptr<TestResult>> Driver::Run() {
 
   for (auto &Test : Finder.findTests(Ctx)) {
     auto ObjectFiles = AllObjectFiles();
-    ExecutionResult ExecResult = Runner.runTest(Test.get(), ObjectFiles);
+
+    ExecutionResult ExecResult = Sandbox.run([&](ExecutionResult *SharedResult){
+      *SharedResult = Runner.runTest(Test.get(), ObjectFiles);
+    });
+
     auto BorrowedTest = Test.get();
     auto Result = make_unique<TestResult>(ExecResult, std::move(Test));
 
     for (auto Testee : Finder.findTestees(BorrowedTest, Ctx)) {
       auto ObjectFiles = AllButOne(Testee->getParent());
       for (auto &MutationPoint : Finder.findMutationPoints(MutationOperators, *Testee)) {
+        ExecutionResult R = Sandbox.run([&](ExecutionResult *SharedResult){
+          /// TODO: here the clone of Testee->getParent() will be used very soon instead.
+          /// For now we are applying mutation to the module same as of mutation point.
+          MutationPoint->applyMutation(Testee->getParent());
+
+          auto Mutant = Compiler.CompilerModule(Testee->getParent());
+          ObjectFiles.push_back(Mutant.getBinary());
 
-        /// TODO: here the clone of Testee->getParent() will be used very soon instead.
-        /// For now we are applying mutation to the module same as of mutation point.
-        MutationPoint->applyMutation(Testee->getParent());
+          /// No longer need to revert mutations since forked child dies:
+          /// ~~Rollback mutation once we have compiled the module~~
+          /// MutationPoint->revertMutation();
```

Comment:
Also rollback of mutations may not be possible, since executing the mutation on a language with global side effects can leak state from one mutation into the next one. So +1 for this change. (but I'd remove commented code).